src/main/java/org/opensearch/sdk/handlers/EnvironmentSettingsResponseHandler.java

```diff
@@ -43,4 +59,8 @@ public String executor() {
     public EnvironmentSettingsResponse read(StreamInput in) throws IOException {
         return new EnvironmentSettingsResponse(in);
     }
+
+    public Settings getEnvironmentSettings() {
```

Comment:
This feels really weird having a handler with a getter, but I don't see any other easy way to do it at the moment.